numpy/core/src/multiarray/compiled_base.c

```diff
@@ -112,7 +112,15 @@ arr_bincount(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
     }
     len = PyArray_SIZE(lst);
 
-    if (mlength == Py_None) {
+    if (mlength == NULL) {
+        minlength = 0;
+    }
+    else if (mlength == Py_None) {
+        /* NumPy 1.14, 2017-06-01 */
```

Comment:
Can you add a note here that once the deprecation warning is removed, one can use `O|On` in the parse call?